pkg/odo/cli/preference/set.go

```diff
@@ -93,10 +84,18 @@ func NewCmdSet(name, fullName string) *cobra.Command {
 		Use:   name,
 		Short: "Set a value in odo config file",
 		Long:  fmt.Sprintf(setLongDesc, preference.FormatSupportedParameters()),
-		Example: fmt.Sprintf(fmt.Sprint("\n", setExample), fullName,
-			preference.UpdateNotificationSetting, preference.NamePrefixSetting,
-			preference.TimeoutSetting, preference.BuildTimeoutSetting, preference.PushTimeoutSetting,
-			preference.ExperimentalSetting, preference.ConsentTelemetrySetting),
+		Example: func(exampleString, fullName string) string {
+			cfg, _ := preference.New()
+			properties := preference.NewPreferenceList(*cfg)
+			for _, property := range properties.Items {
+				value := property.Default
+				if value == "" {
+					value = "smh"
```

Comment:
What's `smh`? I see it show up in example for `odo preference set NamePrefix smh`. Maybe use something simpler?

Reply:
Yeah, I had my doubts about it :sweat_smile: